go get fails

Running the `go get` command yields following message:
```log
$ go version
go version go1.16.2 linux/amd64
$ go get -u github.com/james-bowman/sparse/...
# github.com/james-bowman/sparse/blas
../../go/pkg/mod/github.com/james-bowman/sparse@v0.0.0-20210223104155-34b61a8a5923/blas/matrix.go:83:7: undefined: floats.EqualWithinAbs
../../go/pkg/mod/github.com/james-bowman/sparse@v0.0.0-20210223104155-34b61a8a5923/blas/matrix.go:105:8: undefined: floats.EqualWithinAbs
```
### Expected
For go to get module. I did not have this problem `go get`ing `gonum`.

--- a/blas/matrix.go
+++ b/blas/matrix.go
@@ -1,7 +1,7 @@
 package blas
 
 import (
-	"gonum.org/v1/gonum/floats"
+	"gonum.org/v1/gonum/floats/scalar"
 	"gonum.org/v1/gonum/mat"
 )
 
@@ -80,7 +80,7 @@ func (m *SparseMatrix) insert(i int, j int, v float64, insertionPoint int) {
 func (m *SparseMatrix) nnzWithin(epsilon float64) int {
 	count := 0
 	for _, v := range m.Data {
-		if !floats.EqualWithinAbs(v, 0, epsilon) {
+		if !scalar.EqualWithinAbs(v, 0, epsilon) {
 			count++
 		}
 	}
@@ -102,7 +102,7 @@ func (m *SparseMatrix) Cull(epsilon float64) *SparseMatrix {
 		for minor := startIdx; minor < endIdx; minor++ {
 			col := m.Ind[minor]
 			v := m.Data[minor]
-			if !floats.EqualWithinAbs(v, 0, epsilon) {
+			if !scalar.EqualWithinAbs(v, 0, epsilon) {
 				newInd[curPos] = col
 				newData[curPos] = v
 				curPos++

--- a/cholesky.go
+++ b/cholesky.go
@@ -3,7 +3,7 @@ package sparse
 import (
 	"math"
 
-	"github.com/gonum/floats"
+	"gonum.org/v1/gonum/floats"
 	"gonum.org/v1/gonum/mat"
 )
 

--- a/compressed_test.go
+++ b/compressed_test.go
@@ -3,7 +3,7 @@ package sparse
 import (
 	"testing"
 
-	"gonum.org/v1/gonum/floats"
+	"gonum.org/v1/gonum/floats/scalar"
 	"gonum.org/v1/gonum/mat"
 )
 
@@ -458,7 +458,7 @@ func TestCSTrace(t *testing.T) {
 		for i := 0; i < test.s; i++ {
 			checkTr += m.At(i, i)
 		}
-		if !floats.EqualWithinAbs(tr, checkTr, 1e-13) {
+		if !scalar.EqualWithinAbs(tr, checkTr, 1e-13) {
 			t.Logf("trace mismatch: %f vs %f", tr, checkTr)
 			t.Fail()
 		}

--- a/diagonal_test.go
+++ b/diagonal_test.go
@@ -3,7 +3,7 @@ package sparse
 import (
 	"testing"
 
-	"gonum.org/v1/gonum/floats"
+	"gonum.org/v1/gonum/floats/scalar"
 	"gonum.org/v1/gonum/mat"
 )
 
@@ -148,7 +148,7 @@ func TestDIATrace(t *testing.T) {
 		for i := 0; i < test.s; i++ {
 			checkTr += dia.At(i, i)
 		}
-		if !floats.EqualWithinAbs(tr, checkTr, 1e-13) {
+		if !scalar.EqualWithinAbs(tr, checkTr, 1e-13) {
 			t.Logf("trace mismatch: %f vs %f", tr, checkTr)
 			t.Fail()
 		}

--- a/go.mod
+++ b/go.mod
@@ -3,8 +3,6 @@ module github.com/james-bowman/sparse
 go 1.14
 
 require (
-	github.com/gonum/floats v0.0.0-20181209220543-c233463c7e82
-	github.com/gonum/internal v0.0.0-20181124074243-f884aa714029 // indirect
 	golang.org/x/exp v0.0.0-20210220032938-85be41e4509f
 	gonum.org/v1/gonum v0.8.2
 )

